src/VSSetup.PowerShell/VSSetup.PowerShell.csproj

```diff
@@ -112,7 +114,8 @@
     </PropertyGroup>
     <Error Condition="!Exists('..\..\packages\MicroBuild.Core.0.3.0\build\MicroBuild.Core.props')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\MicroBuild.Core.0.3.0\build\MicroBuild.Core.props'))" />
     <Error Condition="!Exists('..\..\packages\MicroBuild.Core.0.3.0\build\MicroBuild.Core.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\MicroBuild.Core.0.3.0\build\MicroBuild.Core.targets'))" />
-    <Error Condition="!Exists('..\..\packages\Nerdbank.GitVersioning.2.3.38\build\Nerdbank.GitVersioning.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\Nerdbank.GitVersioning.2.3.38\build\Nerdbank.GitVersioning.targets'))" />
+    <Error Condition="!Exists('..\..\packages\Microsoft.VisualStudio.Setup.Configuration.Interop.3.0.4492\build\Microsoft.VisualStudio.Setup.Configuration.Interop.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\Microsoft.VisualStudio.Setup.Configuration.Interop.3.0.4492\build\Microsoft.VisualStudio.Setup.Configuration.Interop.targets'))" />
+    <Error Condition="!Exists('..\..\packages\Nerdbank.GitVersioning.3.4.244\build\Nerdbank.GitVersioning.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\Nerdbank.GitVersioning.3.4.244\build\Nerdbank.GitVersioning.targets'))" />
```

Comment:
Was there an actual change you needed? There were some behavioral changes between v2 and v3. Unless there are changes you need (or security issues, which for this build-time component is unlikely), it's safer to keep the version there.

Reply:
No I will revert this, thanks for pointing that out.

Reply:
I thought you were going to revert this. I wouldn't take the change if not upgrading for good reason.

Reply:
we are updating 3rd party packages because during our security review the guidance is that any 3rd party open source libraries are to be updated to the current versions to be both a good example, and to ensure that we have latest bug fixes.

Reply:
I am going to make sure the Setup reference is on the version it was before because that is not a 3rd party package and as you pointed out it is for backwards compatibility.